contracts/evoting/types/admin.go

```diff
@@ -44,6 +45,51 @@ func (af AdminForm) Deserialize(ctx serde.Context, data []byte) (serde.Message,
 	return message, nil
 }
 
+// AddAdmin add a new admin to the system.
+func (adminForm *AdminForm) AddAdmin(userID string) error {
+	sciperInt, err := strconv.Atoi(userID)
+	if err != nil {
+		return xerrors.Errorf("Failed to convert SCIPER to an INT: %v", err)
+	}
+
+	adminForm.AdminList = append(adminForm.AdminList, sciperInt)
+
+	return nil
+}
+
+// IsAdmin return the index of admin if userID is one, else return -1
+func (adminForm *AdminForm) IsAdmin(userID string) int {
+	sciperInt, err := strconv.Atoi(userID)
```

Comment:
it seems to be only called by `RemoveAdmin`, I think it'd be preferable to cast it to int there, and pass the correct type here

this has the following advantages:

* the error gets checked sooner (the sooner the better) in the execution
* you don't have to cast again here (as you have to cast it in `RemoveAdmin` anyway)